logaut/backends/lydia/_lydia_utils.py

```diff
@@ -46,7 +46,7 @@ def postprocess_lydia_output(output: str) -> str:
     :return: the output associated to the DFA.
     """
     regex = re.compile(
-        r"(?<=Computed automaton:\n).*(?=\n\[2)", flags=re.MULTILINE | re.DOTALL
+        r"(?<=Computed automaton:\r\n).*(?=^\[2)", flags=re.MULTILINE | re.DOTALL
```

Comment:
doesn't this break the one without docker? Maybe add `?` after `\r`
```suggestion
        r"(?<=Computed automaton:\r?\n).*(?=^\[2)", flags=re.MULTILINE | re.DOTALL
```

Reply:
another solution to the problem is to understand why `\r\n` on Docker; it shouldn't be the case

Reply:
You're right. Maybe this would be a better idea. I'm happy to see that also logaut has tests.

Reply:
I'll double check with the docker image